src/test/java/guitests/KeyBindingsGuiTest.java

```diff
@@ -14,6 +14,8 @@
 
 /**
  * Tests key bindings through the GUI
+ * Keybindings tests will fail in travis ci headfull
+ * One magic solution is to click on the listview first before firing shortcuts.
```

Comment:
This explanation is good enough for those of us who know what happened, but not for a new developer. Comments should be written with new developers in mind, not as 'note to self'.